politeia.go

```diff
@@ -18,6 +18,7 @@ type Politeia struct {
 	ctx                     context.Context
 	cancelSync              context.CancelFunc
 	client                  *politeiaClient
+	lastSyncedTimestamp     int64
```

Comment:
This value should be saved to the db. The last synced time will be `0` if the app hasn't done a successful politeia sync.